gradle-baseline-java-config/resources/idea/intellij-java-palantir-style.xml

```diff
@@ -44,8 +44,10 @@
               <package name="" withSubpackages="true" static="false" />
             </value>
           </option>
+          <!-- Should be a superset of https://github.com/google/error-prone/blob/c481b3f9c2da112db36ccfcbf64e755261a127ab/core/src/main/java/com/google/errorprone/bugpatterns/BadImport.java#L63 -->
           <DO_NOT_IMPORT_INNER>
             <CLASS name="Builder" />
+            <CLASS name="BuilderFactory" />
```

Comment:
Looks like this has been out of sync for a while 

https://github.com/google/error-prone/blame/c481b3f9c2da112db36ccfcbf64e755261a127ab/core/src/main/java/com/google/errorprone/bugpatterns/BadImport.java#L66